vuepress/docs/next/tutorials/devops/local-tips-and-tricks.md

```diff
@@ -0,0 +1,146 @@
+---
+sidebarDepth: 2
+--- 
+# Local Development Tips and Tricks
+Here you'll find a list of tips and tricks for local development environments. Please feel free to 
+provide your own or to discuss others on the [forum](https://forum.entando.org). 
+
```

Comment:
Here, you'll find a list of tips and tricks that we've found that can help you optimize development for a local install. We also invite you to our forums to ask questions, collaborate with the community, and discuss tips on how to streamline development of apps on Kubernetes.